src/transactions/index.d.ts

```diff
@@ -0,0 +1,160 @@
+declare module 'neon-js' {
+  type Balance = {
+    GAS: BalanceToken
+    NEO: BalanceToken
+    address: string
+    net: Net
+  }
+  type BalanceToken = {
+    balance: number
+    unspent: BalanceTokenCoin[]
+  }
+  type BalanceTokenCoin = {
```

Comment:
This should be following the JSDoc name of `Coin`. Look at `api.js`. In fact, this is defined in that document instead of this. Might need shuffling. I am also not sure where this lives.

Reply:
Done. Indeed I was afraid that some collusion could happen later.